app/_hub/kong-inc/openid-connect/_index.md

```diff
@@ -462,7 +463,7 @@ params:
       default:
         - sub
       datatype: array of string elements
-      description: 'The claim used to derive a virtual credential (for instance, for the rate-limiting plugin), in case the Consumer mapping is not used.'
+      description: 'The claim used to derive a virtual credential (e.g. to be consumed by the rate-limiting plugin), in case the Consumer mapping is not used.'
```

Comment:
```suggestion
      description: 'The claim used to derive virtual credentials (e.g. to be consumed by the rate-limiting plugin), in case the consumer mapping is not used.'
```